src/main/java/org/gridsuite/cases/importer/job/SftpCaseAcquisitionJob.java

```diff
@@ -0,0 +1,88 @@
+            cassandraProperties.load(Thread.currentThread().getContextClassLoader().getResourceAsStream("cassandra.properties"));
+        }
+
+        Properties sftpProperties = new Properties();
+        try {
+            sftpProperties.load(new FileInputStream("/conf/sftp-server.properties"));
+        } catch (FileNotFoundException e){
+            sftpProperties.load(Thread.currentThread().getContextClassLoader().getResourceAsStream("sftp-server.properties"));
+        }
+
+        Properties sftpCredentialsProperties = new Properties();
+        try {
+            sftpCredentialsProperties.load(new FileInputStream("/.conf/sftp-credentials.properties"));
+        } catch (FileNotFoundException e){
+            sftpCredentialsProperties.load(Thread.currentThread().getContextClassLoader().getResourceAsStream("sftp-credentials.properties"));
+        }
+
+        Properties serviceProperties = new Properties();
+        try {
+            serviceProperties.load(new FileInputStream("/conf/case-import-service.properties"));
+        } catch (FileNotFoundException e){
+            serviceProperties.load(Thread.currentThread().getContextClassLoader().getResourceAsStream("case-import-service.properties"));
+        }
+
+        final SftpConnection sftpConnection = new SftpConnection();
+        final CaseImportServiceRequester caseImportServiceRequester = new CaseImportServiceRequester(serviceProperties.getProperty("service.url"));
+        final CaseImportLogger caseImportLogger = new CaseImportLogger();
+
+
+        try {
+            sftpConnection.open(sftpProperties.getProperty("hostname"), sftpCredentialsProperties.getProperty("user.name"), sftpCredentialsProperties.getProperty("password"));
+            caseImportLogger.connectDb(cassandraProperties.getProperty("cassandra.contact-points"), Integer.parseInt(cassandraProperties.getProperty("cassandra.port")));
+
+            String casesDirectory = sftpProperties.get("cases.directory").toString();
+            List<Path> filesToAcquire = sftpConnection.listFiles(casesDirectory);
+            for (Path file : filesToAcquire) {
+                TransferableFile acquiredFile = sftpConnection.getFile(file.toString());
+                if (!caseImportLogger.isImportedFile(acquiredFile.getName(), sftpProperties.getProperty("label"))) {
+                    System.out.println("Import of : \"" + file.toString() + "\"");
+                    boolean importOk = caseImportServiceRequester.importCase(acquiredFile);
+                    if (importOk) {
+                        caseImportLogger.logFileAcquired(acquiredFile.getName(), sftpProperties.getProperty("label"), new Date());
+                    }
+                } else {
+                    System.out.println("File already imported : \"" + file.toString() + "\"");
+                }
+            }
+        } catch (Exception exc) {
+          System.out.println("Job execution error: " + exc.getMessage());
```

Comment:
Use logger instead.

Reply:
Agreed

Reply:
Logger is now used. Logs generated by sshj library are also now visible.